src/components/Home/ContainerOne/styles.module.scss

```diff
@@ -0,0 +1,18 @@
+.cont1 {
+    width:100%;
+    height: 70%;
+    display: flex;
+    align-items: flex-end;
+}
+
+.containerHome {
+    height: auto;
+    width: 40%;
+    font-size: 29px;
```

Comment:
Te recomendaría usar `28px` o bien `30px`, acostumbramos a usar números pares por consistencia en el código ya que hace más fácil dividir sobre ellos en caso de ser necesario. Te dejo un artículo que habla también sobre [la importancia de usar valores múltiplos de 8](https://iamsteve.me/blog/entry/a-guide-to-vertical-rhythm).

Reply:
Ok, gracias Kike !